packages/backend-common/src/config.ts

```diff
@@ -18,8 +18,40 @@ import { resolve as resolvePath } from 'path';
 import parseArgs from 'minimist';
 import { Logger } from 'winston';
 import { findPaths } from '@backstage/cli-common';
-import { Config, ConfigReader, JsonValue } from '@backstage/config';
-import { loadConfig } from '@backstage/config-loader';
+import { loadConfigSchema, loadConfig } from '@backstage/config-loader';
+import { AppConfig, Config, ConfigReader, JsonValue } from '@backstage/config';
+
+import { setRedactionMap } from './logging';
+
+// Fetch the schema and get all the secrets to pass to the rootLogger for redaction
+const updateRedactionMap = async (configs: AppConfig[], logger: Logger) => {
+  // Consider all packages in the monorepo when loading in config
+  const { Project } = require('@lerna/project');
+  const project = new Project();
+  const packages = await project.getPackages();
+  const localPackageNames = packages.map((p: any) => p.name);
+
+  const schema = await loadConfigSchema({ dependencies: localPackageNames });
```

Comment:
It's enough to just list the local package name, `loadConfigSchema` will detect the rest. I actually just added a new option here that lets you link to a `package.json` with a path, which I think is even better in this case.

Reply:
I've set this to `loadConfigSchema({ dependencies: ['@backstage/backend-common'] })`. I'm not 100% sure that's correct, but please let me know if not!